packages/notification/channel/desktop.ts

```diff
@@ -90,7 +90,7 @@ export const send = async (
 function psEscape(str: string): string {
   let result = ''
   for (const ch of str) {
-    if (ch.charCodeAt(0) === 39) {
+    if (ch.codePointAt(0) === 39) {
       // single quote, escape it with another single quote
       result += ch
     }
```

Comment:
**Suggestion:** The 'psEscape' function uses 'codePointAt' to get the Unicode code point value of a character. This is a good improvement over 'charCodeAt', which only returns a Unicode value in the BMP (Basic Multilingual Plane). This means 'codePointAt' can handle characters outside the BMP, while 'charCodeAt' cannot.
```suggestion
function psEscape(str: string): string {
  let result = ''
  for (const ch of str) {
    if (ch.codePointAt(0) === 39) {
      // single quote, escape it with another single quote
      result += ch
    }
  }
  return result
}
```

Reply:
This suggests to remove `result += ch` after if statement brackets which  is necessary, so I am also resolving this without any changes.